app/controllers/issues_controller.rb

```diff
@@ -58,6 +58,7 @@ def update
 
   def show
     @issue = Issue.find_by_volume(params[:id])
+    @article_revisions = ArticleRevision.joins(:submission).where(submissions: { issue: id })
```

Comment:
To powinno być w osobnej akcji `show_comments` zdefiniowanej w tym kontrolerze. Na wzór akcji takich jak `show_reviews` etc.